html/pages/deluser.inc.php

```diff
@@ -14,7 +14,7 @@
             $delete_username = dbFetchCell('SELECT username FROM users WHERE user_id = ?', array($vars['id']));
 
             if ($vars['confirm'] == 'yes') {
-                if (deluser($delete_username)) {
+                if (deluser($vars['id']) >= 0) {
```

Comment:
Yes, because when you call deluser you will never want to know the user id you deleted.
Basically, there is a hidden rule that you need to add `>= 0` after this method everytime it is called.

I understand that it is only called once and there is three implementations...  Can't fix everything at once.